docs/package.json

```diff
@@ -2,8 +2,8 @@
   "scripts": {
     "dev": "vuepress dev .",
     "build": "NODE_ENV=production vuepress build .",
-    "lint": "prettier --check \"**/*.{js,md,json}\"",
-    "lint:fix": "prettier --write \"**/*.{js,md,json}\""
+    "lint": "prettier --prose-wrap=never --check \"**/*.{js,md,json}\"",
```

Comment:
I know we like to say that we use prettier without any configuration, but now that we've got two pieces of config information (prose-wrap and filetypes) that are identical across all of the packages, I'm wondering if it wouldn't be nice to simplify things and just use a config file.